src/EnergyPlus/EvaporativeCoolers.cc

```diff
@@ -3831,9 +3832,21 @@ namespace EvaporativeCoolers {
 
 		if ( MyFanFlag( UnitNum ) ) {
 			if ( ZoneEvapUnit( UnitNum ).ActualFanVolFlowRate != AutoSize ) {
+
+				if ( ZoneEvapUnit( UnitNum ).ActualFanVolFlowRate < ZoneEvapUnit( UnitNum ).DesignAirVolumeFlowRate ) {
+					ShowSevereError( "InitZoneEvaporativeCoolerUnit: ZoneHVAC:EvaporativeCoolerUnit = " + ZoneEvapUnit( UnitNum ).Name );
+					ShowContinueError( "...unit fan volumetric flow rate less than evaporative cooler unit design supply air flow rate." );
+					ShowContinueError( "...fan volumetric flow rate = " + TrimSigDigits( ZoneEvapUnit( UnitNum ).ActualFanVolFlowRate, 5 ) + " m3/s." );
+					ShowContinueError( "...evap cooler unit volumetric flow rate = " + TrimSigDigits( ZoneEvapUnit( UnitNum ).DesignAirVolumeFlowRate, 5 ) + " m3/s." );
+					ZoneEvapUnit( UnitNum ).DesignAirVolumeFlowRate = ZoneEvapUnit( UnitNum ).ActualFanVolFlowRate;
+					ShowContinueError( "...evaporative cooler unit design supply air flow rate will match fan flow rate and simulation continues." );
+					MyEnvrnFlag( UnitNum ) = true; // re-initialize to set mass flow rate and max mass flow rate
+				}
+
 				if ( ZoneEvapUnit( UnitNum ).ActualFanVolFlowRate > 0.0 ) {
 					ZoneEvapUnit( UnitNum ).DesignFanSpeedRatio = ZoneEvapUnit( UnitNum ).DesignAirVolumeFlowRate / ZoneEvapUnit( UnitNum ).ActualFanVolFlowRate;
```

Comment:
@rraustad Now that the fan is forced to match, DesignFanSpeedRatio can go away.  And the I/O ref should be changed under ZoneHVAC:EvapCooler "Design Supply Air Flow Rate" where it says - This value is typically consistent with the maximum air flow rate of the fan, but it can be less if desired.

Reply:
They are not forced to match, the fan is forced to have a flow greater than or equal to the evap cooler. What if a user hard sizes a fan flow rate greater than the evap cooler? I had just now considered deleting this variable and setting a local FanSpeedRatio = 1 to use in the fan call but then thought of this scenario.

    Fans::SimulateFanComponents( ZoneEvapUnit( UnitNum ).FanName, false, ZoneEvapUnit( UnitNum ).FanIndex, ZoneEvapUnit( UnitNum ).DesignFanSpeedRatio, ZoneCompTurnFansOn, ZoneCompTurnFansOff );

Reply:
I see, well if they don't match it's still broken for the fan:onoff case. The DesignFanSpeedRatio should *never* be applied to the mass flow rate.  It should only be passed to Fan:OnOff when it's called for the full flow case.  I'll send you the changes I was going to put in before I saw this waiting over here.  You can review and see if you agree.

Reply:
FanSpeedRatio should be passed to the fan based on ratio of mass flow rate to max fan flow rate. So there are multiple cases here for max/min primary flow, sec flow on or off, etc. FanSpeedRatio should be calculated for each case.

Reply:
But it's only used by Fan:OnOff.  For the variable flow control Fan:OnOff isn't legal.  And for the places where the system is off, sending a zero makes sure the fan is off.

Reply:
Wait a minute, this isn't a multispeed fan, so there is only one speed and therefore only one speed ratio. So yes I would be interested in seeing what changes you have in mind.

Reply:
Yes, if it's a VAV fan then SpeedRatio doesn't matter.